controllers/drplacementcontrol.go

```diff
@@ -33,11 +33,11 @@ const (
 )
 
 var (
-	WaitForAppResouceRestoreToComplete error = errorswrapper.New("Waiting for App resources to be restored...")
-	WaitForVolSyncDestRepToComplete    error = errorswrapper.New("Waiting for VolSync RD to complete...")
-	WaitForSourceCluster               error = errorswrapper.New("Waiting for primary to provide Protected PVCs...")
-	WaitForVolSyncManifestWorkCreation error = errorswrapper.New("Waiting for VolSync ManifestWork to be created...")
-	WaitForVolSyncRDInfoAvailibility   error = errorswrapper.New("Waiting for VolSync RDInfo...")
+	WaitForAppResourceRestoreToComplete error = errorswrapper.New("Waiting for App resources to be restored...")
+	WaitForVolSyncDestRepToComplete     error = errorswrapper.New("Waiting for VolSync RD to complete...")
+	WaitForSourceCluster                error = errorswrapper.New("Waiting for primary to provide Protected PVCs...")
+	WaitForVolSyncManifestWorkCreation  error = errorswrapper.New("Waiting for VolSync ManifestWork to be created...")
+	WaitForVolSyncRDInfoAvailibility    error = errorswrapper.New("Waiting for VolSync RDInfo...")
```

Comment:
It is very bad that we cannot make one line fix without changing lines around it.

Can we avoid it by using larger indent so future changes in the names will not cause
unwanted change in other lines?

Is this enforced by by gofmt or by 3rd party linter?